tools/accuracy_checker/openvino/tools/accuracy_checker/metrics/language_modeling.py

```diff
@@ -15,6 +15,7 @@
 """
 
 import numpy as np
+from scipy.special import log_softmax
```

Comment:
please make it optional for users who do not want to use this metric, please see as other packages wrapper into UnsupportedPackage if not present as hard dependency

Reply:
done